src/mmw/mmw/settings/layer_settings.py

```diff
@@ -193,13 +193,14 @@
             'display': 'USGS Subbasin unit (HUC-8)',
             'short_display': 'HUC-8 Subbasin',
             'table_name': 'boundary_huc08',
-            'helptext': 'HUC stands for hydrologic unit code. The hydrologic '
-                        'unit hierarchy is indicated by the number of digits in '
-                        'groups of two (such as HUC-2, HUC-4, and HUC-6) within '
-                        'the HUC code. HUC 8 maps the subbasin level, analogous '
-                        'to medium-sized river basins. There are about 2200 '
-                        'nationwide. This tool allows you to pick a predefined '
-                        'HUC-8 to analyze.',
+            'helptext': 'US Geological Survey Hydrologic Units of the '
+                        'eight-digit level (Hydrologic Unit Code 8), averaging'
+                        ' 700 sq. mi. (1,813 km<sup>2</sup>).<br />'
```

Comment:
For each of these, consider using a `nbsp` between `1,813` and `km` (or no space at all)

They wrap weird.
![screen shot 2017-07-20 at 8 43 07 am](https://user-images.githubusercontent.com/7633670/28418581-2fbe73dc-6d2a-11e7-92fe-fad67f11dfc6.png)